shim/opentelemetry-opencensus-shim/src/opentelemetry/shim/opencensus/_patch.py

```diff
@@ -0,0 +1,55 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
```

Comment:
Nit, please use `from x import y` to be consistent unless strictly needed to use `import` directly.

Reply:
Huh I didn't realize that was a convention, what's the reason?

Reply:
It's a nit, the rest of the imports are made in the `from x import y` form and its less repeated code since the module/package does not appear along every symbol.

Reply:
done